packages/@lwc/engine-core/src/framework/weak-multimap.ts

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
I'm not asking for it to be done in this PR, but I'm wondering whether the `engine-core/src/framework` folder is due for reorganizing into smaller sub-folders, grouped by functions/sub-modules. Was thinking this file would go great in a `utils/` folder.

Reply:
Yes it has become a hodgepodge!